qemu/tests/multi_nics_verify.py

```diff
@@ -116,6 +117,12 @@ def _check_ip_number():
 
     # Check all the interfaces in guest get ips
     session_srl = vm.wait_for_serial_login(timeout=int(params.get("login_timeout", 360)))
+    nics_num_checking_cmd = params.get("nics_num_checking_cmd")
+    while True:
+        status, nics_num_checking_cmd_out = session.cmd_status_output(nics_num_checking_cmd, timeout=360)
+        if int(nics_num_checking_cmd_out) == nics_num:
+            break
+        time.sleep(30)
```

Comment:
```suggestion
    utils_misc.wait_for(lambda: int(session.cmd__output(nics_num_checking_cmd, timeout=360)) == nics_num, ***, 30)
```

Reply:
We need to setup an end condition for the while block, otherwise it might get stuck in an endless loop.

Reply:
Done. I will test its result ASAP.